client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -26,7 +26,7 @@ const shouldNext = action => {
 	}
 
 	// is a network response, don't reissue
-	if ( data.data || data.error || data.headers ) {
+	if ( data.data || data.error || data.headers || data.progress ) {
```

Comment:
We can simplify this to the following, but I'm also fine with the current approach.
```jsx
return ! ( data.data || data.error || data.headers || data.progress );
```